sso/sso70-https.json

```diff
@@ -145,15 +145,15 @@
             "displayName": "SSO Admin Username",
             "description": "SSO Server admin username",
             "name": "SSO_ADMIN_USERNAME",
-            "value": "admin",
-            "required": false
+            "value": "",
+            "required": true
         },
         {
             "displayName": "SSO Admin Password",
             "description": "SSO Server admin  password",
             "name": "SSO_ADMIN_PASSWORD",
-            "value": "admin",
-            "required": false
+            "value": "",
```

Comment:
These should probably be generated and some text added to the message telling the user what the uid/password are.  Theoretically, both user and password could be generated.